staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -432,6 +433,10 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 					// has a semantic that it returns data at least as fresh as provided RV.
 					// So first try to LIST with setting RV to resource version of last observed object.
 					klog.V(4).Infof("%s: watch of %v closed with: %v", r.name, r.expectedTypeName, err)
+				case apierrors.IsTooManyRequests(err):
+					klog.V(4).Infof("%s: watch of %v returned 429 - backing off", r.name, r.expectedTypeName)
```

Comment:
not sure about verbosity here... this seems like it should be more visible than an expired error

Reply:
That makes sense - it shouldn't be that frequent. Switched to V(2)